documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -5,66 +5,202 @@
 [id="additional-install-config-parameters_{context}"]
 = Additional `install-config` parameters
 
-This topic describes the required parameters, the `hosts` parameter, 
+This topic describes the required parameters, the `hosts` parameter,
 and the `bmc` `address` parameter for the `install-config.yaml` file.
 
 .Required parameters
-
 |===
 |Parameters |Default |Description
+
+
+
+
+| [[basedomain]] `baseDomain`
+|
+| The domain name for your cluster, e.g. `example.com`
+
+
+a|[[metadataname]]
+----
+metadata:
+    name:
+----
+|
+|The name to be given to your {product-title} cluster. e.g. `openshift`
+
+
+a|[[machinecidr]]
+----
+networking:
+    machineCIDR:
+----
+|
+|The public CIDR (Classless Inter-Domain Routing) of your external network, e.g. `10.0.0.0/24`
+
+
+
+a|[[workername]]
+----
+compute:
+  - name: worker
+----
+|
+|The {product-title} cluster requires a name be provided for compute nodes even if no compute nodes are to be used.
+
+
+a|[[computereplicas]]
+----
+compute:
+    replicas: 2
+----
+|
+|Replicas sets the number of compute nodes that are to be included as part of your {product-title} cluster.
+
+
+
+a|[[controlplanename]]
+----
+controlPlane:
+    name: master
+----
+|
+|The {product-title} cluster requires a name be provided for control plane (master) nodes.
+
+
+a|[[controlplanereplicas]]
+----
+controlPlane:
+    replicas: 3
+----
+|
+|Replicas sets the number of control plane (master) nodes that are to be included as part of your {product-title} cluster.
+
+a|[[baremetalplatform]]
+----
+platform:
+    baremetal: {}
+----
+|
+| FIXME
```

Comment:
Maybe we remove this platform: baremetal piece all together. Not sure how this piece works exactly.

Reply:
removed